README.md

```diff
@@ -60,7 +60,7 @@ repositories {
 
 Add a dependency like
 ```gradle
-implementation(group: 'bio.terra', name: 'terra-workspace-manager-client', version: 'x.x.x')
+implementation(group: 'bio.terra', name: 'workspace-manager-client', version: 'x.x.x')
```

Comment:
what's the easiest way to get the latest version? is it from the bumpbot commit? like [0.254.192](https://github.com/DataBiosphere/terra-workspace-manager/commit/11a92e80ac66a86608dcefbd17e2a15440be0507)?

Reply:
The latest version should always be in [`settings.gradle`](https://github.com/DataBiosphere/terra-workspace-manager/blob/main/settings.gradle#L29), the bumptagbot commits are what keep that file updated

Reply:
Yes, the tag, release, and published library all use the same version.